lib/ace/ext/papeeria/highlighter.coffee

```diff
@@ -97,7 +97,36 @@ define((require, exports, module) ->
                 result.mismatch = false
         return result
 
+    higlightInitBind = (editor) ->
+        session = editor.getSession()
+        keyboardHandler = 
+            name: 'highlightBrackets'
+            bindKey: {win: 'Ctrl-Shift-9', mac: 'Command-Shift-9'}
+            exec: (editor)  -> return highlightBrackets(editor)
+            readOnly: true
+
+
+        editor.commands.addCommand(keyboardHandler);
+
+        session.getSelection().on("changeCursor", () -> 
+            if (session.$bracketHighlightLeft || session.$bracketHighlightRight) 
```

Comment:
in CoffeScript parentheses around `if` condition are optional, and we normally do not use them